addons/mail/models/mail_activity.py

```diff
@@ -504,6 +504,25 @@ def action_feedback_schedule_next(self, feedback=False):
             'target': 'new',
         }
 
+    def _prepare_next_activity_values(self):
```

Comment:
Organization: would move it at the end of file, under a 
```
# ----------------------------------------------------------------------
# TOOLS
# ----------------------------------------------------------------------
```
section so that it does not mess with current list of main action methods. Purpose is to keep main API on top.